operator/config/default/kustomization.yaml

```diff
@@ -19,8 +19,6 @@ bases:
 # [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix including the one in
 # crd/kustomization.yaml
 - ../webhook
-- ../metrics
```

Comment:
I guess this was removed by accident, that would explain why integration tests are failing

Reply:
thanks for noticing this pr needs a rebase :)